### PR TITLE
Sidhant sriv

### PR DESCRIPTION
**Pull request by sidhant-sriv**

### CODE REVIEW

**Review by mynameismon — CHANGES_REQUESTED** [review]
Update your docstrings to reflect the use of the problem

File: Problem-No-23/Approach-No-1.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 string = input("Enter a sentence...\n")
 numbers = '1234567890'
-chars = '!@#$%^&*()<>?:-"\'}+=_{|\][;//.,`~'
+chars = '!@#$%^&*()<>?:-\"\'}+=_{|\][;//.,`~'
```
Comment (mynameismon) [inline Problem-No-23/Approach-No-1.py:9]: Smart.

**Review by mynameismon — APPROVED** [review]
Make sure you update the rest of the docstrings by **today**

**Comment by sidhant-sriv** [comment]
finally

**Pull request merged**